src/main/java/org/jabref/model/database/BibDatabaseContext.java

```diff
@@ -198,7 +198,13 @@ public boolean isBiblatexMode() {
 
         // 4. BIB file directory
         getDatabasePath().ifPresent(dbPath -> {
-            String parentDir = dbPath.getParent().toAbsolutePath().toString();
+            assert dbPath != null : "dbPath is null";
```

Comment:
We do not use `assert` statements in our code. We always do `Objects.requireNonNull(...)`. Is it possible to change that?